packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.tsx

```diff
@@ -23,7 +23,7 @@ export interface AutocompleteProps<T> {
   width?: string;
   renderInput?: (params: AutocompleteRenderInputParams) => React.ReactNode;
   renderOption?: AutocompleteOptionRenderer<T>;
-  value?: AutocompleteOption<T>;
+  value?: AutocompleteOption<T> | null;
```

Comment:
api, I'm guessing? and it means we'll cross the controlled/uncontrolled error boundary at some point?

Reply:
Ah sorry, means we can pass `null` when no value exists (When we've yet to fetch it) rather than `undefined`, which makes the component uncontrolled